build/docker/package.sh

```diff
@@ -14,21 +16,27 @@ for pkgtype in $PKGTYPES ; do
         -f \
         -s dir \
         -t $pkgtype \
-        -n datadog-dotnet-apm \
+        -n otel-dotnet-autoinstrumentation \
```

Comment:
I was hoping that some of these renames could be done via a single local, in this case, we could define "short prefix" and "long prefix" and apply them as needed. I suspect that it won't be possible to do it for all type of files but we could build on top of that by updating the `tools` to do the renames in a single pass. For now, the vendors would have to run the tool but in principle we could even make it part of the build to minimize the boiler-plate divergence.

Reply:
Do we want to do that in this PR or should we create separate task for that?

Reply:
Separate will be better you can look at these changes to see if everything is covered.